### PR TITLE
fix public IP binding

### PR DESCRIPTION
**Pull request by haitch**
the Service Public IP annotation only works on creation time.

`config/prow/k8s/app/svc-envoy.yaml` and `config/prow/k8s/base/contour.yaml` have duplicated content, so removed `svc-envoy.yam`, and added the content to `contour.yaml`.

### CODE REVIEW

File: config/prow/k8s/app/svc-envoy.yaml
Comment (m-chau) [inline config/prow/k8s/app/svc-envoy.yaml:1]: Why is the file removed?

Comment (m-chau) [reply config/prow/k8s/app/svc-envoy.yaml:1]: Oh! It seems like the content is moved to `config/prow/k8s/base/contour.yaml`?

File: config/prow/k8s/app/svc-envoy.yaml [hunk 1]
```diff
@@ -1,23 +0,0 @@
...
-    protocol: TCP
-    targetPort: 8080
-  - port: 443
-    name: https
-    protocol: TCP
-    targetPort: 8443
-  selector:
-    app: envoy
-  type: LoadBalancer
-  loadBalancerIP: $PUBLIC_IP_ADDRESS
```
Comment (m-chau) [inline config/prow/k8s/app/svc-envoy.yaml:23]: "loadBalancerIP: $PUBLIC_IP_ADDRESS" is missing in new edit in `contour.yaml`, but I guess it is ok if the branch is successfully deployed?

Comment (m-chau) [reply config/prow/k8s/app/svc-envoy.yaml:23]: @haitch  would have another PR to address this .

**Review by m-chau — APPROVED** [review]

File: config/prow/k8s/app/svc-envoy.yaml (hunk 1 above)
Comment (haitch) [reply config/prow/k8s/app/svc-envoy.yaml:23]: the IP will be solved by cloud-provider, the issue I am fixing is permission for cloud-provider.

File: config/prow/k8s/app/svc-envoy.yaml
Comment (haitch) [reply config/prow/k8s/app/svc-envoy.yaml:1]: it was duplicated, but the thing is the annotation appending doesn't work, it has to be there on creation time, so I merge them to one place.

**Pull request merged**